packages/formation/js/accordion.js

```diff
@@ -124,6 +124,22 @@ const addAccordionClickHandler = () => {
             if (!isElementInViewport(accordionButton)) {
               element.scrollIntoView();
             }
+
+            // Fire event for subscription by any consuming apps that need to
+            // handle click further (e.g., for analytics).
+            const accordionClickEvent = document.createEvent('Event');
+            accordionClickEvent.initEvent(
```

Comment:
It looks like these methods are [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent). Is it possible to use the [`Event` constructor](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event)?

Reply:
@U-DON , unfortunately no.  I just pushed a code-comment up about this...

Even though createEvent is deprecated, IE11 support requires we keep using it, per the [customEvent doc-page](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent#Browser_compatibility): 
<img width="1300" alt="customEvent-ie11-incompatible" src="https://user-images.githubusercontent.com/587583/63372417-e9d3c480-c33a-11e9-9688-6de8d78ffa35.png">